app/build.gradle

```diff
@@ -87,55 +66,118 @@ android {
             enableSplit = false
         }
     }
+
+    compileOptions {
+        encoding = 'UTF-8'
```

Comment:
Does this need to be specified explicitly? I also don't think there's any place where Unicode characters appear in Java code (unless I snuck an emoji in somewhere)